components/org.wso2.carbon.identity.discovery/src/main/java/org/wso2/carbon/identity/discovery/builders/ProviderConfigBuilder.java

```diff
@@ -110,10 +109,8 @@ public OIDProviderConfigResponse buildOIDProviderConfig(OIDProviderRequest reque
 
         providerConfig.setSubjectTypesSupported(new String[]{"pairwise"});
 
-        providerConfig.setCheckSessionIframe(IdentityUtil.resolveURL(IdentityUtil.getProperty(
-                IdentityConstants.OAuth.OIDC_CHECK_SESSION_EP_URL), true, false));
-        providerConfig.setEndSessionEndpoint(IdentityUtil.resolveURL(IdentityUtil.getProperty(
-                IdentityConstants.OAuth.OIDC_LOGOUT_EP_URL), true, false));
+        providerConfig.setCheckSessionIframe(buildUrl(IdentityConstants.OAuth.OIDC_CHECK_SESSION_EP_URL, null));
```

Comment:
Rather than passing null. It would be better to have an overloaded method with a single argument.